client/src/components/CreateList/CreateList.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useRef } from 'react';
+import {
+  Container,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+  IconButton,
+  InputBase,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import AddIcon from '@material-ui/icons/Add';
+
+import DropZone from './DropZone';
+import FormSubmitButton from '../FormSubmitButton';
+import CloseIconButton from '../CloseIconButton';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(4),
+    paddingLeft: theme.spacing(12),
+    paddingRight: theme.spacing(12),
+
+    height: '70rem',
+  },
+  dialogPaper: {
+    minHeight: '74rem',
+    maxHeight: '74rem',
+    minWidth: '32rem',
+    maxWidth: '72rem',
+  },
+}));
+
+function CreateList() {
+  const [open, setOpen] = useState(false);
+  const [name, setName] = useState('');
+
+  const dropZoneFilesRef = useRef(null);
+  const dropZoneSetFilesRef = useRef(null);
+
+  const classes = useStyles();
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    // eslint-disable-next-line no-unused-vars
+    const formData = {
+      name,
+      cover: dropZoneFilesRef.current[0],
+    };
+    /* TODO: SUBSCRIBE TO REDUX STORE AND PASS FORMDATA TO ACTION CREATORS */
+    setName('');
+    dropZoneSetFilesRef.current([]);
+  };
+
+  return (
+    /* TODO: NEED TO FIX THE ADDICON BUTTON BELOW TO BE DISPLAYED AS A CARD */
+    <div>
+      <IconButton onClick={handleClickOpen} color="primary">
+        <AddIcon />
+      </IconButton>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="form-dialog-title"
+        classes={{ paper: classes.dialogPaper }}
```

Comment:
Obviously this works. I am curious how this is different than setting className={classes.dialogPaper}?

Reply:
I took a closer look at the documentation and played around with the **className** prop in codesandbox. The [documentation](https://material-ui.com/customization/components/#1-specific-variation-for-a-one-time-situation) states that

> Every component provides a className property which is always applied to the root element

So the following

`className={ classes.dialogPaper }`

would apply the styles to the root element, which in this case is the Dialog button. However, this isn't what we want because it is the Paper component that controls the width and height of the container in the middle. 

With that out of the way, note that there is no Paper component inside our code, so how do we access it? The [Dialog API](https://material-ui.com/api/dialog/#css) has a CSS rule for paper that applies styles to it, and the way to pass our styles to it is through the **classes** property.

`classes={{ paper: classes.dialogPaper }}`